docs/guides-agent-app.md

```diff
@@ -0,0 +1,485 @@
+
+Once you've located your Agent and Voting app addresses, run the following command:
+
+```
+dao acl create <your organisation name> <your agent app address> EXECUTE_ROLE <your voting app address> <your voting app address> --environment aragon:rinkeby
+```
+
+You should see the following output:
+
+```
+✔ Generating transaction
+  ✔ Sending transaction
+ ✔ Successfully executed
+```
+If you've reached this stage, 😊🎉 Congratulations! 🎉😊 You've successfully given your Voting app permissions to execute actions on behalf of your Agent app!
+
+Why did that work 😕?
+
+Before we explain the `dao acl create` command we ran above we need to understand a little bit about how permissions in Aragon work.
+
+Aragon uses an [Access Control List](https://hack.aragon.org/docs/acl-intro) (ACL) to control who can access your app's functionality.
+
+This list contains a set of who has permission to execute an action in an Aragon app and who can re-grant or revoke that permission.
+
+[`dao acl create`](https://hack.aragon.org/docs/cli-dao-commands#dao-acl-create) is just the Aragon command used to create a permission in the ACL.
+
+It takes 5 arguments:
+
+1. The name or main address of the DAO
+2. The address of the app whose - permissions are being managed (WHO).
+3. The identifier or name of the role (WHERE).
+4.  The address of the app (or entity) that is being granted the permission (WHAT).
+5. The address of the app (or entity) that will be able to grant that permission or revoke it (HOW).
+
+Let's revisit an annotated version of the command we ran above:
+
+```
+dao acl create
+1. <your organization name>
+2. <your agent app address> 
+3. EXECUTE_ROLE
+4. <your voting app address>
+5. <your voting app address>
+--environment aragon:rinkeby
+```
+You should see that in our case:
+
+- **1** is the name of our organization. 
+
+- **2** is our organization's Agent app -- we are managing the permissions of our Agent app by allowing the Voting app to execute actions on behalf of it.
+
+- **3** is the EXECUTE_ROLE -- The EXECUTE_ROLE is a role defined in the Agent app: it allows an entity to execute a specific action through the Agent app (transferring tokens for example).
+
+- **4** is our organization's Voting app -- we are granting this role to our Voting app, this allows it to execute actions on behalf of our Agent app.
+
+- **5** is our Voting app again -- we are assigning it as manager of the role, this allows it to grant or revoke the permissions of this role.
+
+**Note that, same as before, this command will trigger a vote in the DAO and you'll need to vote *yes* to confirm the new permissions you've granted the Voting app.**
+
+You can do this either by using the UI again or, now that you know how to get the address of your apps, by running:
+
+```
+dao exec <your organization name> <your voting app address> vote 1 true true --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
```

Comment:
~The vote id might not be 1 if there has already been activity in the Voting app~

Edit: I see that this is mentioned below, but this command could have the vote id as a placeholder instead of the `1` directly

Reply:
```suggestion
dao exec <your organization name> <your voting app address> vote <vote id> true true --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
```

Reply:
I've changed this one (because we explain how to find the vote id from the UI just below). But I think in some of the other cases, it makes sense to write the value directly (followed by a comment explaining why we are inputting said value). I don't feel too strongly about this though. Let me know what you think once you've had another look!